src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface OperationHandler {
+    List<FruitTransaction> getFruitFinalQuantity(List<FruitTransaction> list,
+                                                 List<FruitTransaction> listOfFruits);
```

Comment:
The method name `getFruitFinalQuantity` is somewhat misleading as it suggests retrieving a quantity value, but the method returns a list of `FruitTransaction` objects. Consider renaming the method to better reflect its purpose, such as `applyTransactions` or `processTransactions`, which indicates that it's processing a list of transactions to produce a final state.